plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1408,7 +1408,17 @@ function woocommerce_show_product_loop_sale_flash() {
 	 */
 	function woocommerce_get_product_thumbnail( $size = 'woocommerce_thumbnail', $attr = array(), $placeholder = true ) {
 		global $product;
-
+		
+		if( ! is_array( $attr ) )
+		{
+			$attr = array();
+		}
+		
+		if( ! is_bool( $placeholder ) )
+		{
```

Comment:
```suggestion
```

Reply:
My habits to construct code differently - sorry. Improved.